src/_guides/language/language-tour.md

```diff
@@ -983,66 +983,62 @@ For more information about maps, see
 [Generics](#generics) and
 [Maps](/guides/libraries/library-tour#maps).
 
-### Runes
+<a id="characters"></a>
+### Runes and grapheme clusters
 
-In Dart, runes are the UTF-32 code points of a string.
+In Dart, [runes][] are the UTF-32 code points of a string.
```

Comment:
Nitpick: Use "Unicode code points" or "UTF-32 code units" (preferably the former). 

Code points are abstract integers with no size. UTF-32 is an encoding of those into machine numbers, like UTF-8 and UTF-16, except that the *code unit* is a 32-bit integer, which again means that each code point can be represented as one code unit.